src/replication.c

```diff
@@ -55,6 +55,7 @@ int cancelReplicationHandshake(int reconnect);
 void replicationSteadyStateInit(void);
 void setupMainConnForPsync(connection *conn);
 void dualChannelSyncHandleRdbLoadCompletion(void);
+static void fullSyncWithPrimary(connection *conn);
```

Comment:
This function is used in the dual-chan replication path only so I would think it makes more sense to prefix it too
```suggestion
static void dualChanFullSyncWithPrimary(connection *conn);
```